test/functional/inspec_exec_json_test.rb

```diff
@@ -99,12 +86,12 @@
 
     _(out.stderr).must_equal ""
 
-    skip_windows!
     assert_exit_code 0, out
   end
 
   it "flags profile with correct status when not supported" do
-    out = inspec("exec " + File.join(profile_path, "skippy-profile-os") + " --reporter json --no-create-lockfile")
+    profile = File.join(profile_path, "skippy-profile-os")
+    out = inspec("exec " + profile + " --reporter json --no-create-lockfile")
```

Comment:
`out = inspec("exec " + profile + " --reporter json --no-create-lockfile")` is duplicated a bunch here, should we dry that up?

Reply:
I will file tickets on these but I think they're out of scope for this PR for now.